CONTRIBUTING.md

```diff
@@ -2,34 +2,42 @@
+Before you're able to contribute to this repository, you need to sign a
+Contributor License Agreement (CLA). You can follow the links below to
+fill out the appropriate CLA (individual, or corporate):
 
-Please fill out either the individual or corporate Contributor License Agreement
-(CLA).
-
-* If you are an individual writing original source code and you're sure you own
-  the intellectual property, then you'll need to sign an [individual
-  CLA](https://developers.google.com/open-source/cla/individual).
-* If you work for a company that wants to allow you to contribute your work,
-  then you'll need to sign a [corporate
-  CLA](https://developers.google.com/open-source/cla/corporate).
+* **[Individual
+  CLA](https://developers.google.com/open-source/cla/individual):** You are an individual writing original source code and own the intellectual property.
+* **[Corporate
+  CLA](https://developers.google.com/open-source/cla/corporate):** You work for a company that allows you to contribute your work.
 
 Follow either of the two links above to access the appropriate CLA and
 instructions for how to sign and return it. Once we receive it, we'll be able to
 accept your pull requests.
 
-## Contributing A Patch
+## Contributing a patch
 
-1. Submit an issue describing your proposed change to the repo in question.
-1. The repo owner will respond to your issue promptly.
-1. If your proposed change is accepted, and you haven't already done so, sign a
-   Contributor License Agreement (see details above).
-1. Fork the desired repo, develop and test your code changes.
+1. Submit an issue describing your proposed changes.
+1. The repo owner will respond to your issue promptly. Once accepted:
+1. Sign a Contributor License Agreement (see details above) if you haven't done so.
+1. Fork the repo, develop and test your code changes.
 1. Ensure that your code adheres to the existing style in the sample to which
-   you are contributing. Refer to the
-   [Google Cloud Platform Samples Style Guide]
-   (https://github.com/GoogleCloudPlatform/Template/wiki/style.html) for the
-   recommended coding standards for this organization.
+   you are contributing.
 1. Ensure that your code has an appropriate set of unit tests which all pass.
 1. Submit a pull request.
+
+## Samples requirements
+
+All new code sample needs the following requirements:
+- A GitHub Action workflow that tests the sample code. At minimum, this should
+  dry-run any container image or Terraform configs and pass without any errors.
+  [[Example](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/.github/workflows/security-ci.yml)]
+- Cloud Build configs for all container images that pushes to the 
+  `google-samples` artifact registry.
+  [[Example](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/security/wi-secrets/cloudbuild.yaml)]
+- A Terraform section for the above Cloud Build configs, if applicable.
```

Comment:
**Thought (feel free to ignore):**
This process is spelled out in the ["Add Automation for New Samples"](https://docs.google.com/document/d/1Jg8VL5kCn8c8pqBBF7B2FIrilVVfbUU1NPxESwnDlgQ/edit?resourcekey=0-e_m_lbNxVSOWidjfE56MNQ#heading=h.ldcgmvotw7pb) section of a Googler-internal doc.
Feel free to reference. But also, feel free to ignore. Not urgent. :)

Reply:
That's a good point. The documentation should live as close to the source code as possible — ideally, in this repo.
I've [deprecated that Google Doc](https://docs.google.com/document/d/1Jg8VL5kCn8c8pqBBF7B2FIrilVVfbUU1NPxESwnDlgQ/edit?resourcekey=0-e_m_lbNxVSOWidjfE56MNQ#).